CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.12.0] - Unreleased
+
+### Added 
+
+- `source.github_api_endpoint`: override the default API endpoint. This allows you to post commit statuses to repositories hosted by GitHub Enterprise (GHE) instances. When provided, this parameter must be a valid HTTPS URL.
```

Comment:
I would make it more evident that the change adds support for GH Enterprise and that the initial contribution comes from XXX as follows:

```
- Support GitHub Enterprise (see `source.github_api_endpoint` in the README). Initial implementation by @wanderanimrod and @dev-jin.
```

Reply:
Changed as suggested.